target/linux/ramips/image/rt288x.mk

```diff
@@ -33,10 +33,13 @@ endef
 TARGET_DEVICES += ar725w
 
 define Device/f5d8235v1
-  DTS := F5D8235_V1
   IMAGE_SIZE := 7744k
   DEVICE_TITLE := Belkin F5D8235 V1
   DEVICE_PACKAGES := kmod-switch-rtl8366s
+  DEVICE_DTS := F5D8235_V1
+  DEVICE_DTS_DIR := ../dts
```

Comment:
While testing a vanilla bug I have seen various issues getting CONFIG_MIPS_APPENDED_RAW_DTB working at all. Thanks to @mkresin to help me patch the Makefiles for it.

In fact IMHO both lines are required:
Only DEVICE_DTS is used in "Build/append-dtb" of include/image-commands.mk.
If DEVICE_DTS_DIR is not specified the default DTS_DIR pointing to the kernel is tried (which fails). Maybe I missed something where DEVICE_DTS is set to the value of DTS?